index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Microverse</title>
+    <title>Microverse.</title>
 </head>
 <body>
     <h1>Hellow Microverse</h1>
```

Comment:
### Your .gitignore
![Screenshot from 2022-08-23 16-11-08](https://user-images.githubusercontent.com/93039359/186143794-a2c26b53-b4a8-49d0-abdc-ad41e1671704.png)

### Expected .gitignore
![gitignore-fix](https://user-images.githubusercontent.com/93039359/186143808-09a83315-2ffc-4ee4-8101-5e12e65e5b2e.png)

- [ ] Please add your test.md and node_modules in your .gitignore file as shown in the image above :+1: as suggested by the previous reviewer